### PR TITLE
Add info to readme

### PR DESCRIPTION
**Pull request by katrialesser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Run 'grunt' to make sure gulp is installed and available.
 
 Run 'grunt devmode' to watch files, process scss to css and perform other tasks.
 
+### Adding new files
+If you add a partial "\_newfile.scss" 
+inside scss/components folder, it will be included automatically.
```
Comment (buttonwillowsix) [inline README.md]: don't you also need to add it to components.scss?

**Review by buttonwillowsix — CHANGES_REQUESTED** [review]
See comment about partials! I think they do need to be added?

**Comment by buttonwillowsix** [comment]
I will verify this works before merging.

File: README.md (hunk 1 above)
Comment (katrialesser) [reply README.md]: i updated this

**Review by buttonwillowsix — APPROVED** [review]

**Pull request merged**